source/about/cloud-subscriptions.rst

```diff
@@ -0,0 +1,104 @@
+Cloud Subscriptions
+===================
+
+Purchasing a Cloud subscription
+-------------------------------
+
+Mattermost Cloud subscriptions start with a 14-day trial. When you're near the end of your trial, a banner will notify you to upgrade to a subscription plan. 
+
+1. Select **Subscribe now** from the banner
```

Comment:
I believe these numbered items should end in periods

Reply:
Updated.